From 5f143dadc4de094e235deb087537a21e21374e7d Mon Sep 17 00:00:00 2001
From: jbelloua <jean-marie.bellouard@intel.com>
Date: Wed, 6 Nov 2013 10:56:42 +0100
Subject: [PORT FROM MAIN][BT] use same BT FW for VV & FFRD

BZ: 149936

On saltbay PR1, different BT FW were used for FFRD & VV board

Since PR2, same FW is used for both VV & FFRD.
As PR1 are not supported anymore, only one FW will be copied on default FW path

Change-Id: I3d32565371a9bd9ed772ed1cca46cfb3d9691797
Orig-Change-Id: Ic29479c02a9ed59e2cbd6d505e91bedef7d344e0
Category: device enablement
Domain: CWS.BT-Common
Origin: internal
Upstream-Candidate: no, not-aosp
Signed-off-by: jbelloua <jean-marie.bellouard@intel.com>
---
 conf/intel/baylake/bt_vendor.conf     |    2 +-
 conf/intel/byt_m_crb/bt_vendor.conf   |    2 +-
 conf/intel/byt_t_ffrd8/bt_vendor.conf |    2 +-
 conf/intel/ctp7160/bt_vendor.conf     |    2 +-
 conf/intel/saltbay/bt_vendor.conf     |    3 +++
 5 files changed, 7 insertions(+), 4 deletions(-)

diff --git a/conf/intel/baylake/bt_vendor.conf b/conf/intel/baylake/bt_vendor.conf
index e23e986..5755628 100644
--- a/conf/intel/baylake/bt_vendor.conf
+++ b/conf/intel/baylake/bt_vendor.conf
@@ -2,4 +2,4 @@
 UartPort = /dev/ttyMFD0
 
 # Firmware patch file location
-FwPatchFilePath = /etc/firmware/bt/vv/
+FwPatchFilePath = /etc/firmware/bt/
diff --git a/conf/intel/byt_m_crb/bt_vendor.conf b/conf/intel/byt_m_crb/bt_vendor.conf
index e23e986..5755628 100644
--- a/conf/intel/byt_m_crb/bt_vendor.conf
+++ b/conf/intel/byt_m_crb/bt_vendor.conf
@@ -2,4 +2,4 @@
 UartPort = /dev/ttyMFD0
 
 # Firmware patch file location
-FwPatchFilePath = /etc/firmware/bt/vv/
+FwPatchFilePath = /etc/firmware/bt/
diff --git a/conf/intel/byt_t_ffrd8/bt_vendor.conf b/conf/intel/byt_t_ffrd8/bt_vendor.conf
index e23e986..5755628 100644
--- a/conf/intel/byt_t_ffrd8/bt_vendor.conf
+++ b/conf/intel/byt_t_ffrd8/bt_vendor.conf
@@ -2,4 +2,4 @@
 UartPort = /dev/ttyMFD0
 
 # Firmware patch file location
-FwPatchFilePath = /etc/firmware/bt/vv/
+FwPatchFilePath = /etc/firmware/bt/
diff --git a/conf/intel/ctp7160/bt_vendor.conf b/conf/intel/ctp7160/bt_vendor.conf
index 01d8dd2..4940486 100644
--- a/conf/intel/ctp7160/bt_vendor.conf
+++ b/conf/intel/ctp7160/bt_vendor.conf
@@ -2,7 +2,7 @@
 UartPort = /dev/ttyMFD0
 
 # Firmware patch file location
-FwPatchFilePath = /etc/firmware/bt/vv/
+FwPatchFilePath = /etc/firmware/bt/
 
 LPM_IDLE_TIMEOUT_MULTIPLE = 5
 SCO_USE_I2S_INTERFACE = FALSE
diff --git a/conf/intel/saltbay/bt_vendor.conf b/conf/intel/saltbay/bt_vendor.conf
index e6d8154..5755628 100644
--- a/conf/intel/saltbay/bt_vendor.conf
+++ b/conf/intel/saltbay/bt_vendor.conf
@@ -1,2 +1,5 @@
 # UART device port where Bluetooth controller is attached
 UartPort = /dev/ttyMFD0
+
+# Firmware patch file location
+FwPatchFilePath = /etc/firmware/bt/
-- 
1.7.4.1

